web/app/js/components/MetricsTable.jsx

```diff
@@ -198,16 +198,23 @@ export class MetricsTableBase extends BaseTable {
       return { text: ns, value: ns };
     });
 
+    let resource = this.props.resource.toLowerCase();
+
+    let showNsColumn = this.props.showNamespaceColumn;
+    if (resource === "authority" || resource === "namespace") {
```

Comment:
I think we should be able to display the namespace column on the authority list page and have it be meaningful. Am pretty sure that will work with the refactor suggested on #1181.